src/Parser/Mage/Frost/Modules/Features/FreezingRain.js

```diff
@@ -44,22 +44,22 @@ class ArcticGale extends Analyzer {
   statistic() {
     return (
       <StatisticBox
-        icon={<SpellIcon id={SPELLS.ARCTIC_GALE_TALENT.id} />}
+        icon={<SpellIcon id={SPELLS.FREEZING_RAIN_TALENT.id} />}
         value={`${formatPercentage(this.damagePercent)} %`}
-        label="Arctic Gale damage"
-        tooltip="This is the portion of your total damage attributable to Arctic Gale. This number only considers the damage bonus to Blizzard, and does not factor in extra targets you may have hit due to the increased area of effect."
+        label="Freezing Rain damage"
+        tooltip="This is the portion of your total damage attributable to Freezing Rain. This number only considers the damage bonus to Blizzard, and does not factor in extra targets you may have hit due to the increased area of effect."
```

Comment:
no longer increases radius afaik

Reply:
yep, you are right, i didnt see that in the suggestion